README.md

```diff
@@ -1,2 +1,32 @@
-# SSPINN
-Spectral Structure Prediction In Neural Networks
+![SSPINN Logo](./docs/images/SSPINN_Logo.png)
```

Comment:
Can you fix the alignment of the logo? its shifted to the left with whitespace on the right.

Reply:
Yea, absolutely. I need to make it smaller also.